src/components/Home/Navbar.jsx

```diff
@@ -23,23 +25,105 @@ const Bar = styled.div`
 `;
 
 const Links = styled.a`
+  color: white;
   float: left;
   display: block;
-  color: #212121;
   text-align: center;
   padding: 14px 16px;
   text-decoration: none;
   font-size: 17px;
 `;
 
-export const Navbar = ({ setSearch, setSelectedVideo }) => {
+const Text = styled.p`

```

Comment:
If you want to improve the code, you can move the styled to other file, as we saw in the last workshop. This is not a required change, just a suggestion in case you want to give more structure to your code.